style.css

```diff
@@ -39,3 +40,71 @@
   padding-top: 3px;
   padding-right: 5px;
 }
+
+#content {
+  width: 375px;
+  height: 590px;
+  background: #1c1a19;
+  margin: 0;
+  position: relative;
+}
```

Comment:
- [ ] It is required to align the content of this section with `display: flex`. Kindly look into that and fix it.

Reply:
@Qoosim I have aligned the content of this section with `display: flex;`.